libminifi/include/Connection.h

```diff
@@ -52,86 +53,81 @@ class Connection : public core::Connectable {
                       const utils::Identifier &srcUUID, const utils::Identifier &destUUID);
   explicit Connection(std::shared_ptr<core::Repository> flow_repository, std::shared_ptr<core::ContentRepository> content_repo, std::shared_ptr<SwapManager> swap_manager,
                       std::string name, const utils::Identifier& uuid);
-  // Destructor
   ~Connection() override = default;
 
-  // Set Source Processor UUID
+  Connection(const Connection &parent) = delete;
+  Connection &operator=(const Connection &parent) = delete;
+
+  static constexpr uint64_t DEFAULT_BACKPRESSURE_SIZE_THRESHOLD = 10000;
+  static constexpr uint64_t DEFAULT_BACKPRESSURE_DATA_THRESHOLD = 10_MB;
```

Comment:
I would rename these settings, because the data size is also a size. Can we go with backpressure threshold count and backpressure threshold size or backpressure threshold bytes?

Reply:
Good idea :+1: , I've renamed them in https://github.com/apache/nifi-minifi-cpp/pull/1501/commits/8cfacbd1bd5916c7366bbee7e6d86fa916b9d38d